src/main/java/net/coru/api/generator/plugin/OpenapiMultiFileMojo.java

```diff
@@ -282,25 +285,44 @@ private String convertPackageToTargetPath(final String fileSpecPackage, final Bo
   private void processModelsWhenOverWriteIsTrue(
       final FileSpec fileSpec, final OpenAPI openAPI, final String fileModelToSave,
       final List<String> listObjectsToCreate, final String modelPackage,
-      final Map<String, Schema<?>> basicSchemaMap) {
+      final Map<String, Schema<?>> basicSchemaMap) throws TemplateException, IOException {
 
     for (String pojoName : listObjectsToCreate) {
+      final var schemaObject = MapperContentUtil.mapComponentToSchemaObject(openAPI.getComponents().getSchemas(), openAPI.getComponents().getSchemas().get(pojoName),
+                                                                            StringUtils.capitalize(pojoName), fileSpec, modelPackage);
+      checkRequiredOrCombinatorExists(schemaObject);
+
       try {
-        templateFactory.fillTemplateSchema(fileModelToSave, fileSpec.getUseLombokModelAnnotation(),
-                                           MapperContentUtil.mapComponentToSchemaObject(openAPI.getComponents().getSchemas(), openAPI.getComponents().getSchemas().get(pojoName),
-                                                                                        StringUtils.capitalize(pojoName), fileSpec, modelPackage));
+        templateFactory.fillTemplateSchema(fileModelToSave, fileSpec.getUseLombokModelAnnotation(), schemaObject);
       } catch (IOException | TemplateException e) {
         e.printStackTrace();
       }
-      basicSchemaMap.forEach((schemaName, basicSchema) -> {
-        try {
-          templateFactory.fillTemplateSchema(fileModelToSave, fileSpec.getUseLombokModelAnnotation(), MapperContentUtil.mapComponentToSchemaObject(openAPI.getComponents().getSchemas(), basicSchema, schemaName,
-                                                                                                                                                   fileSpec,
-                                                                                                                                                   modelPackage));
-        } catch (IOException | TemplateException e) {
-          e.printStackTrace();
+    }
+    basicSchemaMap.forEach((schemaName, basicSchema) -> {
+      final var basicSchemaObject = MapperContentUtil.mapComponentToSchemaObject(openAPI.getComponents().getSchemas(), basicSchema, schemaName, fileSpec, modelPackage);
+      checkRequiredOrCombinatorExists(basicSchemaObject);
+      try {
+        templateFactory.fillTemplateSchema(fileModelToSave, fileSpec.getUseLombokModelAnnotation(), basicSchemaObject);
+      } catch (IOException | TemplateException e) {
+        e.printStackTrace();
```

Comment:
Shouldn't be better to use some kind of logs here, instead of printStackTrace?

Reply:
It could be a good option, but it is done like this throughout the plugin. We will analyze if it is necessary to create a new issue to deal with it.